src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/organization/config/OrganizationIngestionWFConfig.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.pu.workflow.wf.ingestionflow.organization.config;
+
+import io.temporal.workflow.Workflow;
+import it.gov.pagopa.payhub.activities.activity.ingestionflow.UpdateIngestionFlowStatusActivity;
+import it.gov.pagopa.payhub.activities.activity.ingestionflow.email.SendEmailIngestionFlowActivity;
+import it.gov.pagopa.payhub.activities.activity.ingestionflow.organization.OrganizationIngestionActivity;
+import it.gov.pagopa.pu.workflow.config.temporal.BaseWfConfig;
+import it.gov.pagopa.pu.workflow.config.temporal.TemporalWFImplementationCustomizer;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+  @Configuration
+  @ConfigurationProperties(prefix = "workflow.payments-reporting-ingestion")
```

Comment:
you should also set these properties!
```suggestion
  @ConfigurationProperties(prefix = "workflow.organization-ingestion")
```